rbx_reflection/src/database.rs

```diff
@@ -59,6 +76,15 @@ impl<'a> ReflectionDatabase<'a> {
         Some(list)
     }
 
+    /// Returns an iterator of superclasses for the provided ClassDescriptor. This
+    /// iterator will start with the provided class and end with `Instance`.
+    pub fn superclasses_iter(&'a self, descriptor: &'a ClassDescriptor<'a>) -> SuperClassIter {
+        SuperClassIter {
+            database: self,
+            descriptor,
+        }
```

Comment:
```suggestion
            descriptor: Some(descriptor),
```
I have serious concerns if you did not even attempt to compile this code before submitting it, which is what this type mismatch indicates to me.

Reply:
GitHub appears to have added this suggestion to the wrong line, but hopefully it's obvious what I meant

Reply:
Whoops, I was spewing some nonsense about Rust generators not being stable and got distracted.  Fixed